src/green/mbpt/except.h

```diff
@@ -40,9 +40,5 @@ namespace green::mbpt {
     explicit mbpt_chemical_potential_search_failure(const std::string& what) : std::runtime_error(what) {}
   };
 
-  class mbpt_outdated_input : public std::runtime_error {
```

Comment:
Same problem here, moving error class to `green-integrals` does not make sense, since the context of an error is different. I propose to bring it back.